### ISSUE TITLE
GUI : FED : Table check boxes label accessibility violations

### ISSUE THREAD

**Issue by derick-montague** [ibm-openbmc/dev#2756]
## Overview
The check boxes used for all the tables do not meet accessibility guidelines.

1. The check boxes in the table rows are missing a label value.
![image.png](https://images.zenhubusercontent.com/57e6780495e42b6c5ba963ac/ca7ffbe5-d7d3-4bba-9160-40c1206edab2)

1. The select all check box is missing a label
![image](https://user-images.githubusercontent.com/1183515/98824705-38b2c180-23f9-11eb-8051-b2b7b97d2f8a.png)

### Tasks
- [ ] Run the IBM accessibility checker and resolve all the checkbox accessibility violations
- [ ] Table header label `select all` or `unselect all`
- [ ] Row label `select` or `unselect`
- [ ] Check with IBM language guidelines to determine if term `unselect` or `deselect` is the proper verb
- [ ] Identify and update all tables throughout the application

## UX Flow / Interaction Requirements
N/A

## Final Invision Mockup
N/A

## cURL Commands
N/A

### Browser Tests (Chrome, Firefox, Safari (Mac), Edge (Windows))
- [ ] Tested responsive layout on supported viewport sizes
- [ ] Verified functionality works as expected, e.g. success and failure use cases
- [ ] Resolved any JavaScript errors thrown to the console

### Accessibility Tests
- [ ] Verified markup is semantic
- [ ] No reported issues in IBM Accessibility Checker (Free open source tool) https://www.ibm.com/able/toolkit/tools/

**Comment by dixsie** [comment]
resolved by: https://gerrit.openbmc-project.xyz/c/openbmc/webui-vue/+/38439

**Comment by derick-montague** [comment]
Resolved by https://gerrit.openbmc-project.xyz/c/openbmc/webui-vue/+/38611
